_sass/common.scss

```diff
@@ -141,3 +141,23 @@ header,
 .invisible {
     display: none !important;
 }
+//login scope
+.fx-acc {
```

Comment:
No, I sent new pages for oath (login and scope) where fx-login is renamed to fx-acc so it can be reused.